tox.ini

```diff
@@ -34,17 +37,12 @@ commands =
 [testenv:bbp-model]
 # Please module load neurodamus-neocortex py-neurodamus beforehand
 passenv = *
-deps =
-    -r tests/requirements.txt
 setenv =
     PYTHONPATH={toxinidir}:{env:PYTHONPATH}
     HOC_LIBRARY_PATH={toxinidir}/core/hoc:{env:HOC_LIBRARY_PATH}
     NEURON_INIT_MPI=1
-allowlist_externals =
-    /usr/bin/echo
 commands =
-    /usr/bin/echo {env:PYTHONPATH}
-    /usr/bin/echo {env:HOC_LIBRARY_PATH}
+    # {env:HOC_LIBRARY_PATH}
```

Comment:
The env variable gets expanded, so `echo` will actually make it show twice. I was hoping this to be shown in the shell, but it seems it's taken as a comment. It could stay for future reference, but if you want I can drop it

Reply:
I was just curious

Reply:
actually I found a way. Assigning to a variable will make it clear and not print twice

Reply:
@jorblancoa I tried but tox doesnt really like var assignments :/

I'm reverting

Reply:
@jorblancoa @heerener I need a re-approval

Reply:
Thanks. seems a good idea.